pkg/handler/unrouted_handler.go

```diff
@@ -915,15 +915,8 @@ func (handler *UnroutedHandler) sendError(w http.ResponseWriter, r *http.Request
 		statusErr = NewHTTPError(err, http.StatusInternalServerError)
 	}
 
-	reason := append(statusErr.Body(), '\n')
-	if r.Method == "HEAD" {
-		reason = nil
-	}
-
 	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
```

Comment:
The `http.Error` method automatically sets the same Content-Type header, so could remove this here. See https://golang.org/src/net/http/server.go?s=62104:62156#L2011